source/rules/no-unsafe-first.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import { isCallExpression, isIdentifier, typecheck } from "../utils";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows unsafe first/take usage in Effects and epics.",
+      recommended: false
+    },
+    fixable: null,
+    messages: {
+      forbidden:
+        "Unsafe `first` and `take` usage in Effects and epics are forbidden."
+    },
+    schema: [
+      {
+        properties: {
```

Comment:
Is this OK? The [docs](https://github.com/cartant/rxjs-tslint-rules#rules) say it's not configurable but the [code](https://github.com/cartant/rxjs-tslint-rules/blob/master/source/rules/rxjsNoUnsafeFirstRule.ts#L17) tell it's a string or an array, but I don't see the implementation of it being an array?

Reply:
Yeah. Probably just a copy/paste thing.